charts/weblate/templates/secret.yaml

```diff
@@ -7,9 +7,9 @@ metadata:
 {{ include "weblate.labels" . | indent 4 }}
 type: Opaque
 data:
-  postgresql-user: {{ .Values.postgresql.postgresqlUsername | b64enc | quote }}
-  postgresql-password: {{ .Values.postgresql.postgresqlPassword | b64enc | quote }}
-  redis-password: {{ .Values.redis.password | b64enc | quote }}
+  postgresql-user: {{ "postgres" | b64enc | quote }}
```

Comment:
What is the reason for hard-coding the PostgreSQL user?

Reply:
It's because postgresql user is now created using a boolean. And as it need an admin user i choose to harcode because AFAIK we can't create an admin user using the postgresql chart. There is still a posibility to choose a different username if using an external database with environment variable.

Reply:
I can still pick this from values, to provide flexibility in case there is a posibility to select a different admin user with postgresql chart.

Reply:
Thanks for clarifying. As long as it works for external database setup, it's probably fine (I'm not really expert in helm, so I might be asking stupid questions here).